roles/mariadb/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
+
+# Use the new container infrastructure
+sudo: false
+
+# Install ansible
+addons:
+  apt:
+    packages:
+      - python-pip
+
+install:
+  # Install ansible
+  - pip install ansible
```

Comment:
After this, you print the version number. If you care about the version, you should tell pip to install a version equal or newer than pip. Something like this:
pip install ansible "package>=1.2.3"